remove_files.go

```diff
@@ -2,48 +2,66 @@ package remove
 
 import (
 	"context"
+	"fmt"
 	"io/fs"
 	"os"
 	"path/filepath"
 
 	ignore "github.com/Diogenesoftoronto/go-gitignore"
 )
 
-const RemoveSIPFilesName = "remove-sip-files"
+const RemoveFilesName = "remove-files-activity"
 
-type RemoveSIPFilesActivity struct{}
+type RemoveFilesActivity struct{}
 
-func NewRemoveSIPFilesActivity() *RemoveSIPFilesActivity {
-	return &RemoveSIPFilesActivity{}
+func NewRemoveFilesActivity() *RemoveFilesActivity {
+	return &RemoveFilesActivity{}
 }
 
-type RemoveSIPFilesParams struct {
-	SipPath     string
-	SuccumbPath string
+type RemoveFilesParams struct {
+	RemovePath string // The remove path is the path that files will be removed from.
+	IgnorePath string // The ignore path is the path that contains the ignore/succumb file that configures which files will be removed.
 }
 
-type RemoveSIPFilesResult struct{}
+type RemoveFilesResult struct {
+	Removed []string
```

Comment:
Do we have to worry at this time about payload size limits? e.g. if the limit is 4mb then we could start seeing issues if we're returning 50k entries for example.

Reply:
@sevein I can't think of any real world scenario where we'd be removing even a hundred files from a SIP. The use case that lead to this work is removing a single `.DS_Store` directory accidentally added to the root of a BagIt bag.  If thousands of files are being removed from a SIP, there's probably something wrong.

The only alternatives I can see for providing feedback to an operator on which files were deleted from a SIP would be to write the data to a text file (either in a structured format like XML, or just a plain-text list) or to a database, either of which would require a significant amount of work to show in the Enduro UI.  This gives an operator visibility on the files removed via the Temporal UI "for free", which is nice.